src/background/EventManager.ts

```diff
@@ -112,6 +116,29 @@ export class EventManager {
         const errorMessage = this.app.getErrorMessage(params.tabId);
         sendResponse(errorMessage);
         break;
+      case "getConfiguration":
+        try {
+          const configuration = this.app.getConfiguration();
+          sendResponse(configuration);
+        } catch (error) {
+          sendResponse(error as StorageError);
+        }
+        break;
+      case "setConfiguration":
+        params = message.params as { configuration: Configuration };
+        try {
+          this.app.setConfiguration(params.configuration);
+          sendResponse(true);
+        } catch (error) {
+          sendResponse(error as Error);
+        }
+        break;
+      case "deleteCache":
+        console.log("cache was deleted.");
+        break;
+      case "exportLogs":
+        console.log("logs were exported.");

```

Comment:
The `exportLogs` message is not implemented yet, is this also coming in the next pull request?

Reply:
Yes everything regarding the configuration will be done in a separate branch.